setup-balena-cli/action.yml

```diff
@@ -0,0 +1,22 @@
+name: "balena cli setup"
+author: "@andreavs"
+description: "An action to setup the balena CLI and login"
+branding:
+  icon: "truck"
+  color: "gray-dark"
+inputs:
+  api_key:
+    description: "Balena API key"
+    required: true
+runs:
+  using: "composite"
+  steps:
+    - run: |
+        curl https://github.com/balena-io/balena-cli/releases/download/v14.3.0/balena-cli-v14.3.0-linux-x64-standalone.zip -L -o balena.zip
```

Comment:
Could consider adding cli version as a parameter to the action, then you wouldn't have to change the action if it changes.

Reply:
made it an input with a default value - there is some cost to allowing different versions (especially if there is some break in behavior), but I think that is a good trade-off.